compat/maven-model-builder/src/main/resources/org/apache/maven/model/pom-4.1.0.xml

```diff
@@ -20,8 +20,9 @@ under the License.
 -->
 
 <!-- START SNIPPET: superpom -->
-<project>
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.1.0"
```

Comment:
I'm not sure where we've landed on this, but this is still wrong. The namespace should not be changing with every model version, and the sooner we fix that the better. This is what model versions are for.